lib/windowsbasetest.pm

```diff
@@ -46,7 +47,9 @@ sub select_windows_in_grub2 {
 
 sub open_powershell_as_admin {
     my ($self, %args) = @_;
-    send_key_until_needlematch 'quick-features-menu', 'super-x';
+    my $version = (get_var('VERSION') == '11') ? '11' : '';
```

Comment:
Since you do this 3 times, maybe what about extracting this into a method?
```
sub get_version_tag {
    return check_var('VERSION', '11') ? '11' : '';
}
```

and then  just `my $version = get_version_tag()`

Reply:
Or use a variable.